emrun.py

```diff
@@ -1659,6 +1655,13 @@ def run():
     # to support binding to port zero we must allow the server to open to socket then retrieve the final port number
     options.port = httpd.socket.getsockname()[1]
 
+  if not file_to_serve_is_url:
+    if len(options.cmdlineparams):
+      url += '?' + '&'.join(options.cmdlineparams)
+    hostname = socket.gethostbyname(socket.gethostname()) if options.android else options.hostname
+    # to support binding to port zero we must allow the server to open to socket then retrieve the final port number
```

Comment:
I don't think there is a need to repeat this comment, or at least it should be re-phrased to better apply to this block.

Reply:
I am open to suggestions, this seems appropriate to me